examples/charm++/zerocopy/entry_method_api/prereg/stencil3d/stencil3d.C

```diff
@@ -181,7 +181,7 @@ class Stencil: public CBase_Stencil {
       // set period arbitrarily small so that LB occurs when AtSync is called
       // this is in case the default LBPERIOD is larger than the time to complete LBPERIOD_ITER
       // iterations
-      getLBDB()->SetLBPeriod(0);
+      getLBMgr()->SetLBPeriod(0);
```

Comment:
Shouldn't this and the other similar changes to tests/examples be removed now?

Reply:
Good call, I've taken them out.